### ISSUE TITLE
Character continues moving in a direction with no input

### ISSUE THREAD

**Issue by Nolshine** [ParadiseSS13/Paradise#12900]
**Issue Description**:

Sometimes your avatar will keep walking in a direction you pressed to walk in and won't stop unless you mash various controls.

**What did you expect to happen**: 

Character should stop when direction key is lifted.

**What happened instead**:

Character keeps moving in the same direction.

**Why is this bad/What are the consequences**:

Disrupts gameplay, causes you to go in undesirable directions, prevents you from moving when you are stuck walking into a wall.

**Steps to reproduce the problem**:

Just trying to walk seems to cause it often.

**When did the problem start happening**:

I've noticed the problem in the last week or two.

**Extra information**:

I use hotkeys to move, never use the cursor keys, so I don't know if issue exists for cursor key movement.

**Comment by craftxbox** [comment]
This happens in alot of games. What happens is a KEY_DOWN event is registered and then the KEY_UP occurs when the window is nolonger focused so the app doesnt get the event, thus causing the app to think the key is still being held. personally i think this is a 'wont fix' bug

**Comment by Sheppysneik** [comment]
This happens to me too and in general at the start of every shift once controlling a non-ghost character with the sole difference that pressing again the key does not fix it, instead reconnecting does. The strange thing is that after fixing by reconnecting the problem disappears completely for the rest of the shift. Are you per chance referring to this?

**Comment by Nolshine** [comment]
As Sheppysneik said, pressing the offending key (or a bunch of other common offenders like alt, control, tab, space, etc.) doesn't stop the problem. Can't say I tried reconnecting, I will have to give that a go. Maybe it's because BYOND is very pop-up-window-heavy that this happens frequently in it and not in anything else but I still swear I haven't had it happen at all until quite recently. I don't know what else to say about it. I'm versed enough in common software issues that this struck me as a bug rather than a windows quirk. I ahelped, admin said they were getting that too, and that I should report it on the repo.
